e2e-tests/cypress/fixtures/signals/klokNotOnMap.json

```diff
@@ -8,7 +8,7 @@
     "departments": "VOR",
     "created_by": null,
     "text": null,
-    "handling_time": "21 werkdagen"
+    "handling_time": "21"
```

Comment:
How come this change is necessary, when all other `handling_time` values are postfixed with `werkdagen` or `dagen` ?